freeipa/src/main/java/com/sequenceiq/freeipa/converter/FreeIpaRequestToFreeIpaConverter.java

```diff
@@ -3,13 +3,13 @@
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.stereotype.Component;
 
-import com.sequenceiq.freeipa.api.model.freeipa.FreeIpaRequest;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.FreeIpaServerRequest;
 import com.sequenceiq.freeipa.entity.FreeIpa;
 
 @Component
-public class FreeIpaRequestToFreeIpaConverter implements Converter<FreeIpaRequest, FreeIpa> {
+public class FreeIpaRequestToFreeIpaConverter implements Converter<FreeIpaServerRequest, FreeIpa> {
```

Comment:
rename to FreeIpaServerRequestToFreeIpaConverter to match conversion classes

Reply:
fixed these